src/modules/VegaEncodings.ts

```diff
@@ -167,14 +167,19 @@ const bifrostVegaMark = [
 const minimalEncoding = ['x', 'y', 'color', 'opacity', 'size', 'facet'];
 minimalEncoding.sort();
 
-export const vegaMarkEncodingMap: Record<string, string[]> = {
+export const vegaMarkEncodingMap: { [mark in VegaMark]: string[] } = {
   point: minimalEncoding,
   circle: minimalEncoding,
   square: minimalEncoding,
   tick: minimalEncoding,
   line: minimalEncoding,
   bar: minimalEncoding,
   trail: minimalEncoding,
+  errorband: minimalEncoding,
+  boxplot: minimalEncoding,
+  errorbar: minimalEncoding,
+  area: minimalEncoding,
+  rect: minimalEncoding,
```

Comment:
circle, square, trail, area, and rect should be commented out because they are not in VegaMark